package.json

```diff
@@ -50,6 +50,9 @@
   "dependencies": {
     "@sinonjs/commons": "^2.0.0"
   },
+  "optionalDependencies": {
```

Comment:
@SimenB that's a fair point. Would you mind making a ticket for it, so we don't forget?

Reply:
AFAIK when looking at this, it was `chokidar`. And this was only an issue with Node 14/NPM 6, not with any later version, so I was thinking it was more of an issue of it only being able to handle the v1 of the package-lock format, whereas the package-lock file has v2.

And I think we _want_ all consumers of mac to install this? This package ensures it uses file system events instead of polling.

Reply:
> AFAIK when looking at this, it was `chokidar`. And this was only an issue with Node 14/NPM 6, not with any later version, so I was thinking it was more of an issue of it only being able to handle the v1 of the package-lock format, whereas the package-lock file has v2.

Could we upgrade the version of npm used on CI so the lockfile is supported? Or stay on v1 I guess (unless newer npm just updates the format automatically)

> And I think we _want_ all consumers of mac to install this? This package ensures it uses file system events instead of polling.

`@sinonjs/fake-timers` doesn't do any FS at all, tho, so it shouldn't come with `fsevents` (or `chokidar`).

Reply:
#455